README.md

```diff
@@ -435,7 +433,7 @@ Our Pods don't currently have any resource constraints, so the increasing load w
 
 Let's prevent that by adding some resource constraints to the Deployment:
 
-```
+```yaml
 resources:
   requests:
     memory: "0.5Gi"
```

Comment:
They seem to eventually run out of memory and be restarted. I think the flask dev server leaks memory, but I haven't had a chance to try something else